### PR TITLE
Fix typo in last example in Generics

### PR DESCRIPTION
**Pull request by mightyiam**
I don't know all the syntaxes of TypeScript, but this seems like a typo.

### CODE REVIEW

File: pages/Generics.md [hunk 1]
```diff
@@ -319,9 +319,7 @@ class Lion extends Animal {
     keeper: ZooKeeper;
 }
 
-function findKeeper<A extends Animal, K> (a: {new(): A;
-    prototype: {keeper: K}}): K {
```
Comment (mhegazy) [inline pages/Generics.md]: not sure i see why `;` is a typo?

Comment (mightyiam) [inline pages/Generics.md]: Isn't that `;` inside an object literal, where a `,` should be? And the line drop also sees inconsistent.

Comment (mightyiam) [inline pages/Generics.md]: Or is that not an object literal, but an interface?

Comment (mhegazy) [inline pages/Generics.md]: both `;` and `,` are allowed in object literal types.

```ts
var y: { a: number; b, c: string };
```

**Pull request closed without merging**